packages/genesys-spark-components/src/components/beta/gux-form/gux-form-description/gux-form-description.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, h, JSX } from '@stencil/core';
+
+/**
+ *  @slot - Slot for description.
+ */
+
+@Component({
+  styleUrl: 'gux-form-description.scss',
+  tag: 'gux-form-description',
+  shadow: true
+})
+export class GuxFormDescription {
+  render(): JSX.Element {
+    return (<slot></slot>) as JSX.Element;
```

Comment:
I think because we are using `::slotted(*)` we can't use the default slot. That type of slot will accept a text node which that css selector will not target.
We should discuss this at standup as it has become a common issue.